### ISSUE TITLE
Add another layer to the ppo optimiser

### ISSUE THREAD

**Issue by wd7512** [William-Dennis/meta-learning#15]
This could adjust the number of hidden nodes, learning rate, weight decay...
